rpms/openshift-odo.spec

```diff
@@ -92,6 +93,6 @@ Obsoletes:      %{package_name}-redistributable <= %{odo_version}
 %{_datadir}/%{name}-redistributable/odo-darwin-amd64
 %{_datadir}/%{name}-redistributable/odo-darwin-amd64.tar.gz
 %{_datadir}/%{name}-redistributable/odo-windows-amd64.exe
-%{_datadir}/%{name}-redistributable/odo-windows-amd64.exe.tar.gz
+%{_datadir}/%{name}-redistributable/odo-windows-amd64.exe.gz
```

Comment:
But we're not replace `tar.gz` with `zip` here. It's replaced by `.gz` instead. This is not supposed to be `.zip`, right?

Reply:
Ha, I had .gz, then I made the change to zip but forgot to save. Now it should be .zip